### PR TITLE
chore(ci) de duplicate the dependency pinning logic

### PR DESCRIPTION
**Pull request by hutchic**
Utilize the Kong make / bash to de duplicate the dependency pinning logic. Instead of this repository setting the openresty / luarocks / openssl versions we should be using rely on Kong `make setup-ci` to utilize the dependencies as defined in it's `.requirements` file

### CODE REVIEW

**Comment by eskerda** [comment]
I do not see adding a `Makefile` as a solution, since we would have to go over all the repos that depend on `kong-ci` and change it (and create it), and then maintain it. Let's test a mixed approach, where `kong-ci` uses the downloaded kong to setup the dependencies et all: https://github.com/Kong/kong-ci/pull/13 to minimize the number of things to change all around.

**Comment by hutchic** [comment]
I think approach for this PR depends on scope of the change we're looking to take on at this time.

If we want to locate the dependency pinning in one sole location with the least amount of work updating Kong-ci probably fits the bill. I do FWIW envision some future amalgamation of about 3 repositories into one at which time changing all the plugins may be advisable but I think it's fine to tackle that when we get to it.

TLDR; get kong-ci in such a way that it snags dependencies from git://kong/.requirements working and I'm cool with that :)

**Comment by kidd** [comment]
I think we can close this, right? as we picked another way of solving the issue.
